tf-modules/cumulus-rds-tf/variables.tf

```diff
@@ -0,0 +1,74 @@
+
+variable "aws_db_subnet_group" {
+  description = "Name for RDS database cluster subnet group"
+  type        = string
+  default     = "cumulus-rds-tf-subnet"
+}
+
+variable "apply_immediately" {
+  description = "If true, RDS will apply updates to cluster immediately, instead of in the maintenance window"
+  type        = bool
+  default     = true
+}
+
+variable "backup_window" {
+  description = "Preferred database backup window (UTC)"
+  type        = string
+  default     = "07:00-09:00"
+}
+
+variable "backup_retention_period" {
+  description = "Number of backup periods to retain"
+  type        = number
+  default     = 1
+}
+
+variable "deletion_protection" {
+  description = "Flag to prevent terraform from making changes that delete the database in CI"
+  type        = bool
+  default     = true
+}
+
+variable "cluster_identifier" {
+  description = "DB Itentifier for the RDS cluster that will be created"
+  type        = string
+  default     = "cumulus-rds-serverless-default-cluster"
+}
+
+variable "db_username" {
```

Comment:
I might consider renaming this to "admin_db_username". Same for the password. Just to make it clear how it is used. And update the descriptions accordingly

Reply:
👍  on the descriptions.  That said this module's sole purpose is to setup the cluster, which makes the distinction less useful. 

2ddc06a65, d8f77d458